When trying to locate libblas and liblapack, numpy will use the host
versions of those libraries if available. This breaks compilation when
linking.

diff -Naur numpy-1.4.1/numpy/core/setup.py numpy-1.4.1/numpy/core/setup.py
--- numpy-1.4.1/numpy/core/setup.py	2010-04-22 11:35:23.000000000 +0200
+++ numpy-1.4.1/numpy/core/setup.py	2011-11-16 17:11:22.000000000 +0100
@@ -777,7 +777,7 @@
 
     # Configure blasdot
     blas_info = get_info('blas_opt',0)
-    #blas_info = {}
+    blas_info = {}
     def get_dotblas_sources(ext, build_dir):
         if blas_info:
             if ('NO_ATLAS_INFO',1) in blas_info.get('define_macros',[]):
diff -Naur numpy-1.4.1/numpy/linalg/setup.py numpy-1.4.1/numpy/linalg/setup.py
--- numpy-1.4.1/numpy/linalg/setup.py	2010-04-18 12:06:18.000000000 +0200
+++ numpy-1.4.1/numpy/linalg/setup.py	2011-11-16 17:11:35.000000000 +0100
@@ -9,7 +9,7 @@
     config.add_data_dir('tests')
 
     # Configure lapack_lite
-    lapack_info = get_info('lapack_opt',0) # and {}
+    lapack_info = get_info('lapack_opt',0) and {}
     def get_lapack_lite_sources(ext, build_dir):
         if not lapack_info:
             print "### Warning:  Using unoptimized lapack ###"